parent/pom.xml

```diff
@@ -112,6 +113,21 @@
         <maven.shade.plugin.version>3.2.4</maven.shade.plugin.version>
         <maven.source.plugin.version>3.2.1</maven.source.plugin.version>
         <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
+
+        <!-- We leave it empty so that, when jacoco does not run, the unreplaced @{argLine} does not impede the build. -->
+        <additionalTestArgLine/>
+        <commonTestArgLine>
```

Comment:
I think camelcase doesn't match the style of other properties

Reply:
Renamed the properties, but they still contain a segment called `argLine` because it's how this is called in surefire/failsafe configuration